src/content/api/plugins.md

```diff
@@ -39,7 +39,7 @@ noted.
 ## Plugin Types
 
 Depending on the hooks used and `tap` methods applied, plugins can function in
-a number of different ways. The way this works is closely related to the
+a different number of ways. The way this works is closely related to the
```

Comment:
Is there change in meaning intended here?

I'm no native speaker, but looks like before it said "there are many ways, and it depends which way it will be actually", and now it's says "there may be many or not so many ways, it depends"

Reply:
This is still evaluated the same.

Reply:
FWIW I'm a non-native speaker as well and the previous version of this sentence sounded more understandable to me.

Reply:
To me it sounds like the plugins, before, could work in various ways; but now it sounds like they work in a different amount of ways (???)

Reply:
the previous sounds clearer to me.